docs/langref.md

```diff
@@ -162,6 +162,31 @@ Example := Rectangle {
 }
 ```
 
+You can annotate the properties with a qualifier that specifies how the property can be read and written.
+
+ * **`private`**: The property can only be accessed from within the component.
+ * **`in`**: The property is an input, it can be set and modified by the user of this component,
```

Comment:
```suggestion
 * **`in`**: The property is an input. It can be set and modified by the user of this component,
```

(although I'd leave out the "The property is an input" sentence, I don't think it's quite self-explaining)